app/src/main/res/values-de/strings.txt

```diff
@@ -0,0 +1,309 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="format_minutes">%1$dm</string>
+    <string name="format_hours">%1$dh</string>
+    <string name="format_days">%1$dd</string>
+    <string name="format_time_left">%1$s left</string>
+    <string name="format_db">%1$s dB</string>
+    <string name="format_kbps">%1$s kbps</string>
+    <string name="format_ms">%1$s ms</string>
+
+    <string name="format_go_to_artist">Mehr von %1$s</string>
+    <string name="format_top_playlist">Top %1$s</string>
+    <string name="format_my_top_playlist">Meine Top %1$s</string>
+    <string name="format_view_top_of_header">View top %1$s of …</string>
+    <string name="format_speed_multiplier">%1$sx</string>
+    <string name="format_opening_url">Opening url %1$s</string>
+
+    <string name="past_24_hours">Letzten 24 Stunden</string>
+    <string name="past_week">Innerhalb 1 Woche</string>
+    <string name="past_month">Innerhalb 1 Monats</string>
+    <string name="past_year">Innerhalb 1 Jahres</string>
+    <string name="all_time">Gesamter Zeitraum</string>
+
+    <string name="songs">Songs</string>
+    <string name="playlists">Wiedergabelisten</string>
+    <string name="albums">Albums</string>
+    <string name="artists">Künstler</string>
+    <string name="singles">Singles</string>
+
+    <string name="favorites">Favorites</string>
+    <string name="offline">Offline</string>
+    <string name="overview">Übersicht</string>
+    <string name="library">Bibliothek</string>
+    <string name="discover">Entdecken</string>
+    <string name="local">Lokal</string>
+    <string name="quick_picks">Startseite</string>
+    <string name="mood">Stimmung</string>
+    <string name="online">Online</string>
+    <string name="videos">Videos</string>
+    <string name="featured">Ausgewählt</string>
+    <string name="about">Info</string>
+    <string name="appearance">Erscheinungsbild</string>
+    <string name="colors">Farben</string>
+    <string name="shapes">Formen</string>
+
+    <string name="skip_back">Zurückspringen</string>
+    <string name="skip_forward">Vorwärts springen</string>
+    <string name="play">Abspielen</string>
+    <string name="pause">Pause</string>
+    <string name="like">Like</string>
+    <string name="shuffle">Shuffle</string>
+    <string name="sync">Sync</string>
+    <string name="start_radio">Radio Starten</string>
+    <string name="play_next">Als nächstes wiedergeben</string>
+    <string name="enqueue">Einreihen in die Warteschlange</string>
```

Comment:
I think we make this a bit shorter, if that is possible, because currently, the UI could show it like this on smaller screens:

![image](https://github.com/25huizengek1/ViMusic/assets/50515369/2a2c1ae5-a630-44d1-9997-655cf599b278)

Reply:
> I think we make this a bit shorter, if that is possible, because currently, the UI could show it like this on smaller screens:
> 
> ![image](https://private-user-images.githubusercontent.com/50515369/287216948-2a2c1ae5-a630-44d1-9997-655cf599b278.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE0MjQ2NDAsIm5iZiI6MTcwMTQyNDM0MCwicGF0aCI6Ii81MDUxNTM2OS8yODcyMTY5NDgtMmEyYzFhZTUtYTYzMC00NGQxLTk5OTctNjU1Y2Y1OTliMjc4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjAxVDA5NTIyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTljNTEzMTQ4YTAxN2ExYWI3ZWMxMWRmNDVlMDRjNzUxZWI0ZjE1NjQ0MTlmNTk5MWZlNDQ2MGJmNmU0MjRjMDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.YhKCCHVscKu3BcTuA9G1ZRdyu5rb6HAFh2J1YL_lIh8)

I have corrected it. It should no longer break the ui.

Reply:
Seems good, thanks!